templates/deployment.yaml

```diff
@@ -138,6 +138,32 @@ spec:
             readOnly: true
           {{- end }} 
 
+        - name: {{ .Chart.Name }}-job-scanner
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command: ["bash"]
+          args: 
+          - |
+            oda-api -u DISPATCHER_URL inspect --group-by-job;
```

Comment:
I think what would be good to add is some kind of failsafe, i.e. when this whole script fails the container should not crash. 
Else it can not be deployed and it's annoying. And for an optional feature like this it should get deployed anyway.